projects/gloo/cli/pkg/flagutils/install.go

```diff
@@ -12,6 +12,7 @@ func AddInstallFlags(set *pflag.FlagSet, install *options.Install) {
 	set.StringVarP(&install.HelmChartOverride, "file", "f", "", "Install Gloo from this Helm chart archive file rather than from a release")
 	set.StringSliceVarP(&install.HelmChartValueFileNames, "values", "", []string{}, "List of files with value overrides for the Gloo Helm chart, (e.g. --values file1,file2 or --values file1 --values file2)")
 	set.StringVar(&install.HelmReleaseName, "release-name", constants.GlooReleaseName, "helm release name")
+	set.StringVar(&install.Version, "version", "", "version to install (e.g. 1.2.0, defaults to latest)")
```

Comment:
```suggestion
	set.StringVar(&install.Version, "version", "", "version of Gloo to install (e.g. 1.2.0, defaults to latest)")
```
to disambiguate an invocation like `glooctl install knative --version 1.2.0`

Reply:
Follow up question: Is `glooctl install knative --version ...` still too ambiguous?

Reply:
I think it's fine, @marcogschmidt thoughts?

Reply:
@ilackarms thoughts?

Reply:
It is fine with me, as long as the doc for the flag is clear.